mass_editing.py

```diff
@@ -180,6 +180,11 @@ def _object_hook(self, dct):
 
                 if fields[field.name].get('add_remove'):
                     del fields[field.name]['add_remove']
+            elif field.ttype == 'selection':
+                if hasattr(EditingModel, 'get_'+field.name):
```

Comment:
This is not the right way to check if the selection field uses a method to obtain the list of possible values. You should use:

if isinstance(field.selection, str)

And then obtain the list using:

selection = getattr(EditingModel, field.selection)()